tools/Customization/DevHome.Customization/ViewModels/DevDriveInsights/DevDriveCardViewModel.cs

```diff
@@ -31,6 +39,41 @@ public partial class DevDriveCardViewModel : ObservableObject
 
     public string DevDriveFreeSizeText { get; set; }
 
+    public bool IsDevDriveTrusted { get; set; }
+
+    public char DriveLetter { get; set; }
+
+    private const string Fsutil = "fsutil.exe";
+
+    [RelayCommand]
+    private void MakeDevDriveTrusted()
+    {
+        // Launch a UAC prompt
+        var startInfo = new ProcessStartInfo();
+        startInfo.WindowStyle = ProcessWindowStyle.Hidden;
+        startInfo.FileName = Path.Join(Environment.SystemDirectory, Fsutil);
+
+        // Run the fstutil cmd to trust the dev drive
+        startInfo.Arguments = $"devdrv trust /f {DriveLetter}:";
+        startInfo.UseShellExecute = true;
+        startInfo.Verb = "runas";
+
+        var process = new Process();
+        process.StartInfo = startInfo;
+
+        // Since a UAC prompt will be shown, we need to wait for the process to exit
+        // This can also be cancelled by the user which will result in an exception
+        try
+        {
+            process.Start();
+            process.WaitForExit();
+        }
+        catch (Exception ex)
+        {
+            Log.Error(ex.ToString());

```

Comment:
not sure if `ex.ToString()` captures the stack trace. But I think you might be able to just do `Log.Error(ex, "Unable to trust Dev Drive volume with drive letter '{DriveLetter}'");`